.github/workflows/test.yml

```diff
@@ -38,6 +38,8 @@ jobs:
             - 'features/**'
             - 'podjs/**'
             - 'build/**'
+      - name: Install global modules
+        run: npm ci --no-update-notifier --no-fund
```

Comment:
Since this happens (again) as part of the `lint` command now - no need to do it twice, right?

Reply:
We *do* need it, because lint is only run if one of the lintable files is changed. I'm putting it back here.